build.cake

```diff
@@ -1,5 +1,6 @@
 #tool "nuget:?package=xunit.runner.console"
 #addin nuget:?package=Cake.Git
+#addin "Cake.Docker"
```

Comment:
Does this work for you?  I think I had to pin the versions of various addins before it worked on 19.5

Reply:
It seems to: https://travis-ci.org/syncromatics/Syncromatics.Clients.Metro/builds/280512443

Reply:
Oh, I think I remember now.  Pretty sure I had to downgrade packages which had already been updated for cake 22.  Since that one hasn't been upgraded yet it will work for now, though only until it is upgraded for 22.